internal/controller/controllers/bmh_agent_controller.go

```diff
@@ -729,6 +730,22 @@ func (r *BMACReconciler) reconcileBMH(ctx context.Context, log logrus.FieldLogge
 	return reconcileComplete{dirty: true, stop: true}
 }
 
+func (r *BMACReconciler) isNonePlatform(ctx context.Context, cd *hivev1.ClusterDeployment) (isNone bool, err error) {
```

Comment:
please, document this function.

It may not be obvious to future readers that `UserNetworkManagedNetworking` means None. For example, some may wonder why not checking the [ControlPlaneAgents](https://github.com/openshift/assisted-service/blob/f140a9ed60f1507e2cdeb71092e486918d50b6a6/api/hiveextension/v1beta1/agentclusterinstall_types.go#L284)  field instead to check if it's an SNO cluster. The answer is that we want to be able to deploy HA clusters that are platform None.

Reply:
OK